exec/vector/src/main/java/org/apache/drill/exec/record/metadata/TupleMetadata.java

```diff
@@ -46,6 +46,10 @@
 public interface TupleMetadata extends Propertied, Iterable<ColumnMetadata> {
 
   public static final String IS_STRICT_SCHEMA_PROP = DRILL_PROP_PREFIX + "strict";
+  public static final String HAS_HEADERS_PROP = DRILL_PROP_PREFIX + "headers";
+  public static final String SKIP_FIRST_LINE_PROP = DRILL_PROP_PREFIX + "skipFirstLine";
+  public static final String DELIMITER_PROP = DRILL_PROP_PREFIX + "delimiter";
```

Comment:
Can we pass special symbols in delimiter prop? How for example new line can be passed?
`'drill.delimiter' = '\n'` will work? Or we need to escape it?